### PR TITLE
Break down get methods

### PR DESCRIPTION
**Pull request by chanjonathan**

### CODE REVIEW

File: project/pom.xml [hunk 1]
```diff
@@ -1,111 +1,140 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jetty.version>9.4.41.v20210516</jetty.version>
 
-    <!-- Project-specific properties -->
-    <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
-    <googleCloudProjectId>summer22-sps-22</googleCloudProjectId>
-  </properties>
+        <!-- Project-specific properties -->
+        <exec.mainClass>com.google.sps.ServerMain</exec.mainClass>
+        <googleCloudProjectId>jchan-sps-summer22</googleCloudProjectId>
```
Comment (EvanJuV) [inline project/pom.xml]: Let's try to keep the googleCloudProjectId as summer22-sps-22. You can still deploy to your own website, just remember to change this back before pushing.

File: project/src/main/java/com/google/sps/servlets/DeleteServlet.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        try {
+            database.delete(entryID);
+        } catch (SQLException sqlException) {
+            ServletException servletException = new ServletException(sqlException.getMessage());
+            servletException.setStackTrace(sqlException.getStackTrace());
+            throw servletException;
+        }
+    }
+
+    private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```
Comment (EvanJuV) [inline project/src/main/java/com/google/sps/servlets/DeleteServlet.java]: Given that you're putting this function in all your servlets, you could add it to a shared utility library.

**Review by EvanJuV — CHANGES_REQUESTED** [review]
Nice Jonathan and thanks for the cleanup! Please take a look at my two comments. Please also ping me to review your changes again as I don't read my personal email that often.

File: project/pom.xml (hunk 1 above)
Comment (chanjonathan) [reply project/pom.xml]: fixed

File: project/src/main/java/com/google/sps/servlets/DeleteServlet.java (hunk 2 above)
Comment (chanjonathan) [reply project/src/main/java/com/google/sps/servlets/DeleteServlet.java]: added a utilities/Http class

**Review by EvanJuV — APPROVED** [review]

**Pull request merged**